website/.storybook/main.js

```diff
@@ -0,0 +1,28 @@
+const path = require("path");
+
+module.exports = {
+  stories: [
+    "../src/components/**/*.stories.mdx",
+    "../src/components/**/*.stories.@(js|jsx|ts|tsx)",
+  ],
+  addons: [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions",
+    "@chakra-ui/storybook-addon",
+  ],
+  framework: "@storybook/react",
+  core: {
+    builder: "@storybook/builder-webpack5",
+  },
+  staticDirs: ["../public"],
+  typescript: { reactDocgen: false },
+  webpackFinal: async (config, { configType }) => {
+    config.resolve.modules = [path.resolve(__dirname, ".."), "node_modules"];
+
+    return config;
+  },
+  features: {
+    emotionAlias: false,
```

Comment:
why is this necessary?

Reply:
necessary for for chakra to work properly, see [here](https://chakra-ui.com/getting-started/with-storybook)